widgetssdk/src/main/java/com/glia/widgets/view/head/ChatHeadContract.java

```diff
@@ -31,13 +31,16 @@ interface View extends BaseView<Controller> {
 
         void showQueueing();
 
+        void showScreenSharing();
+
         void showOnHold();
 
         void hideOnHold();
 
         void navigateToChat();
 
         void navigateToCall();
+        void navigateToEndScreenSharing();
```

Comment:
NIT. I would add an empty line above navigateToEndScreenSharing(), to keep formatting consistent. Or remove empty lines everywhere.